SUPPORTED_MODELS.md

```diff
@@ -366,6 +392,7 @@ In addition to image generation models, `InpaintingPipeline` supports specialize
     </tr>
     <tr>
       <td>Distil-Whisper</td>
+      <td><code>Not supported</code></td>
```

Comment:
Sometimes it's wrapped with `<code>` and sometimes not. Stay consistent

Reply:
good catch, fixed.